Datadog/Datadog.xcodeproj/xcshareddata/xcschemes/DatadogCore iOS.xcscheme

```diff
@@ -106,7 +106,7 @@
          </EnvironmentVariable>
          <EnvironmentVariable
             key = "DD_DISABLE_NETWORK_INSTRUMENTATION"
-            value = "$(DD_DISABLE_NETWORK_INSTRUMENTATION)"
+            value = "1"
```

Comment:
probably we should revert

Reply:
no - we want to keep it like that because URLSession instrumentation in our SDK doesn't play well with testing SDK URLSession instrumentation